Updated breadcumb navigation

adamzemmoura opened the pull request:
Added breadcrumb trail to left hand side.
Increased font-size to 20px.
Changed icon from fa-dashboard to fa-home.
Updated each page so that correct name appears in breadcrumb trail.
Home link and current page links now work as expected. 
Added custom_css.css file to apply some custom styling without changing bootstrap file.

**NOTE: Needs updated to handle multi-layer navigation i.e. any links between home and current page if won't currently work**

jmandrade commented on public/css/custom-css.css:
```diff
@@ -0,0 +1,6 @@
+    color: #DB4C3F;
+    content: ">";
+    padding: 0 5px;
+}
```
Trailing newline is needed.

adamzemmoura replied:
Not sure what you mean by a trailing new line?  Added where?  Why?

jmandrade replied:
At the very end of the file. Each file must end on a newline or Github will complain.

adamzemmoura replied:
Ahh okay, thanks!

jmandrade commented:
So, you only made changes to the CSS, it seems. We need to remove the breacrumbs from master.blade.php and they need to be added to each page.

Did you modify the minified bootstrap.min.css file? It's listed under changes, but it's not possible to do code review on a minified file.

adamzemmoura commented:
I tried to update the bootstrap file but realizing I shouldnt be doing that and thought I changed it back... any ideas how I can make 100% it's the way it was? Copy / paste the file from Master?

I did change remove the breadcrumbs from master and added it to each page, think I forgot to push those changes.  They should be there now with the last commit.

jmandrade commented:
Yeah. You may copy from master and replace it, to bring it back (there are other ways like "resetting", but copying is faster).

Besides that, it looks good!

adamzemmoura commented:
Copy and pasted the bootstrap.min.css file from master to make sure it's the correct one.
Added a blank line at EOF.
